apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.YCSB/Program.cs

```diff
@@ -28,6 +37,8 @@ internal class Program : YcsbRunner
         private static readonly string s_cloudspanner_projectName = "cloudspanner.project";
         private static readonly string s_boundedStaleness = "cloudspanner.boundedstaleness";
         private static readonly string s_numchannels = "cloudspanner.numchannels";
+        private static readonly string s_raw = "cloudspanner.raw"; //execute directly bypassing veneer and gapic
```

Comment:
This isn't currently bypassing the gapic/toolkit code - you're still using Google.Cloud.Spanner.V1.SpannerClient. To get the raw gRPC client, use the `SpannerClient.GrpcClient` property to get a `Google.Cloud.Spanner.V1.Spanner+SpannerClient` object (as in, the nested type). The naming is somewhat confusing here, but the hope is that very few users ever need to go down to the gRPC level.